cfde_deriva/metrics.py

```diff
@@ -117,7 +129,7 @@ def register_datapackage_measurements(registry_catalog, submission_id, records):
     by_metric = {}
     for record in records:
         metric = _get_required(record, 'metric')
-        value = _get_required_number(record, 'value')
+        value = _get_optional_number(record, 'value')
```

Comment:
I'm not sure why we would register a metric that has not been evaluted... shouldn't we just skip those?